packages/wrangler/src/api/dev.ts

```diff
@@ -79,6 +79,7 @@ export interface UnstableDevOptions {
 		testScheduled?: boolean; // Test scheduled events by visiting /__scheduled in browser
 		watch?: boolean; // unstable_dev doesn't support watch-mode yet in testMode
 	};
+	experimentalDevenvRuntime?: boolean;
```

Comment:
```suggestion
	experimentalDevEnvRuntime?: boolean;
```

Reply:
No this is how yargs camel cases `--experimental-devenv-runtime` – best to just be consistent and not force it

Reply:
oh, I didn't realise that. kk, carry on then :)